test/test_ecs_utils.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from unittest import mock
+from dcicutils.ecs_utils import ECSUtils
+
+
+@pytest.fixture(scope='module')
+def ecs_utils():
+    return ECSUtils(cluster_name='dummy-cluster')
+
+
+def mock_update_service(*, cluster, service, forceNewDeployment):
+    """ Mock matching the relevant API signature for below (we don't actually want
+        to trigger an ECS deploy in unit testing.
+    """
+    return
+
+
+@pytest.mark.parametrize('service_name', [
+    ECSUtils.WSGI, ECSUtils.INDEXER, ECSUtils.INGESTER
+])
+def test_ecs_utils_basic_should_proceed(ecs_utils, service_name):
+    with mock.patch.object(ecs_utils.client, 'update_service', mock_update_service):
```

Comment:
I might suggest making a `make_update_service_mock` that takes expected values as arguments and returns the function that is the mock. e.g.,
```
def make_update_service_mock(expected_cluster, expected_service, expected_forceNewDeployment):
    def mock_update_service(*, cluster, service, forceNewDeployment):
        assert cluster == expected_cluster
        assert service == expected_service
        assert forceNewDeployment == expected_forceNewDeployment
```
so you can later
```
with mock.patch.object(ecs_utils.client, 'update_service', make_mock_update_service(...)):
```
specifying expectations, etc.